src/main/java/frc/robot/commands/step/SeaCucumber.java

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands.step;
+
+import com.chaos131.auto.ParsedCommand;
+import com.chaos131.swerve.BaseSwerveDrive;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.Constants.LauncherConstants;
+import frc.robot.Constants.LiftConstants;
+import frc.robot.subsystems.Feeder;
+import frc.robot.subsystems.Intake;
+import frc.robot.subsystems.Launcher;
+import frc.robot.subsystems.Lift;
+
+// TODO: Implement actual control logic
+public class SeaCucumber extends Command {
```

Comment:
What in the world?

Reply:
lol it was what Dan called it and the kids were amused by it, so I temporarily allowed calling it that. 
I'm going to push up a change calling it LaunchSpit though (because it spits out on the launcher side)

Reply:
RIP SeaCucumber. Mar 2nd - Mar 5th.